OrbitClientData/ModuleData.cpp

```diff
@@ -67,16 +72,22 @@ void ModuleData::LoadSymbols(const orbit_grpc_protos::ModuleSymbols& module_symb
 }
 
 void ModuleData::ClearSymbols() {
-  CHECK(is_loaded());
-  absl::MutexLock lock(&functions_mutex);
+  absl::MutexLock lock(&mutex_);
 
+  CHECK(is_loaded_);
   functions_.clear();
+  hash_to_function_map_.clear();
   is_loaded_ = false;
 }
 
-// Sorted by address
-std::vector<const FunctionInfo*> ModuleData::GetFunctionList() const {
-  absl::MutexLock lock(&functions_mutex);
+[[nodiscard]] orbit_client_protos::FunctionInfo* ModuleData::FindFunctionFromHash(
```

Comment:
Not sure if I asked previous, but can the be const? If not, please just resolve the comment.

Reply:
I dont know why I this still wasn't const. I made it const and removed the old comment